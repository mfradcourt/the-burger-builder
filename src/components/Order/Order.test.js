import React from 'react';
import { shallow } from "enzyme";

import Order from './Order';

describe('<Order />', () => {
  let wrapper;

  it('should render Order', () => {
    wrapper = shallow(
      <Order/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
