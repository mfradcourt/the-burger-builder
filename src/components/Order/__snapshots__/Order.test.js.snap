// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Order /> should render Order 1`] = `
<div
  className="Order"
>
  <p>
    Ingredients: 
  </p>
  <p>
    Price: 
    <strong>
      USD 
      NaN
    </strong>
  </p>
</div>
`;
